index.html

```diff
@@ -11,49 +11,205 @@
+            <div class="wrapper">
+                <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+                <div class="ic_menu">
+                    <button type="button"><i class="fas fa-bars fa-2x"></i></button>
+                </div>
+            </div>
+        </header>
+        <section class="flex-container">
+            <h1 class="title_name">
+                Charlie
+                Giorgania
+            </h1>
+            <p class="about_description">
+                Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+                works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+            <div class="collaborate_button">
+                <button type="button" class="start-collaboration">Start collaboration</button>
             </div>
-        </div>
-
-    </header>
-
-    <section class="flex-container">
-        <h1 class="title_name">
-            Charlie
-            Giorgania
-        </h1>
-        <p class="about_description">
-            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
-            works.
-            If you like what you see and have a project you need coded, don’t hesitate and contact me.
-        </p>
-        <a><div class="collaborate_button">
-            <p id="start-collaboration" >start collaboration</p>
-        </div>
-        </a>
 
-    </section>
-    <section class="image-container">
-        <div id ="image_geometry_2-1"></div>
-    </section>
+        </section>
+        <section class="image-container">
+            <div id="image_geometry_2-1"></div>
+        </section>
 
-    <section class="project_container">
-        <div class="project_title_holder">
-           <h3 id="project-heading">Project</h3>        
-        </div>
+        <section class="project_container">
+            <div class="section_title_holder">
+                <h3>Projects</h3>
+            </div>
+            <div class="project-grid">
 
+                <div class="grid-item1">
+                    <div class="grid-project-image1"></div>
+                    <div class="project-details">
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>JavaScript</p>
+                            </div>
+                        </div>
```

Comment:
- [ ] The list of project technologies ("JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`